src/components/pages/career-details/career-details-structured-data.tsx

```diff
@@ -20,48 +25,57 @@ const concatDescription = (position: SyPersonioJob): string => {
   );
 };
 
+/**
+ * Create and write structured data based on https://schema.org/JobPosting
+ *
+ * See the Google guide for additional insights:
+ * https://developers.google.com/search/docs/advanced/structured-data/job-posting#structured-data-type-definitions
+ */
 export const CareerDetailsStructuredData = ({
   position,
 }: CareerDetailsStructuredDataProps) => {
-  const employmentType = convertScheduleToEmploymentType(position.schedule);
-
-  const data = {
-    '@context': 'https://schema.org/',
+  const structuredData: WithContext<JobPosting> = {
+    '@context': 'https://schema.org',
     '@type': 'JobPosting',
     title: position.name,
     datePosted: position.createdAt,
     description: concatDescription(position),
     identifier: {
       '@type': 'PropertyValue',
       name: 'Satellytes Digital Consulting GmbH',
-      value: '338504-41230',
+      value: position.jobId,
     },
     hiringOrganization: {
       '@type': 'Organization',
       name: 'Satellytes Digital Consulting GmbH',
       sameAs: 'http://www.satellytes.com',
-      logo: 'https://we-are-hiring.cdn.personio.de/logos/41230/social/6ca8f29950517ac39fcae0d6545a1fe6.png',
```

Comment:
Removed the hotlink to personio plus we should provide structured data for our company logo in general, this is only valid for override by third-party companies.

Reply:
Do we already provide a logo to google? If not, as far as I understood the documentation, this property can also be used to override existing logos of your own company (e.g. when you change your logo, Google detects this prop change earlier than the general logo change.

If Google already has our logo, this property would be only unnecessary in the worst case.

Reply:
Let's tackle this through https://github.com/satellytes/satellytes.com/issues/516